## Replacing the dynamic reconfigure server #
#
# initial current amplitude
initial_current: 8

# max current among all channels
stim_limit: 110

# speed angle shift parameter
shift: 0

# current amplitude relation/multiplier
stim_proportion: {
    'Ch1': 1.00, 'Ch2': 1.00,
    'Ch3': 0.00, 'Ch4': 0.00,
    'Ch5': 0.00, 'Ch6': 0.00,
    'Ch7': 0.00, 'Ch8': 0.00
}

# angle range for each stim channel
Ch12: {
  'Ch1_Angle_Min': 280,
  'Ch1_Angle_Max': 20,
  'Ch2_Angle_Min': 100,
  'Ch2_Angle_Max': 200
}

Ch34: {
  'Ch3_Angle_Min': 30,
  'Ch3_Angle_Max': 105,
  'Ch4_Angle_Min': 210,
  'Ch4_Angle_Max': 285
}

Ch56: {
  'Ch5_Angle_Min': 70,
  'Ch5_Angle_Max': 170,
  'Ch6_Angle_Min': 250,
  'Ch6_Angle_Max': 350
}

Ch78: {
  'Ch7_Angle_Min': 280,
  'Ch7_Angle_Max': 20,
  'Ch8_Angle_Min': 100,
  'Ch8_Angle_Max': 200
}
